src/components/ProfilePage/ProfileAvatar.jsx

```diff
@@ -64,49 +64,51 @@ class ProfileAvatar extends React.Component {
     if (this.props.isDefault) {
       return (
         <Button
-          className="text-white btn-block"
-          color="link"
-          size="sm"
+          className={'text-white btn-block btn-sm'.split(' ')}
+          buttonType="link"
           onClick={this.onClickUpload}
-        >
-          <FormattedMessage
-            id="profile.profileavatar.upload-button"
-            defaultMessage="Upload Photo"
-            description="Upload photo button"
-          />
-        </Button>
+          label={
+            <FormattedMessage
+              id="profile.profileavatar.upload-button"
+              defaultMessage="Upload Photo"
+              description="Upload photo button"
+            />
+          }
+        />
       );
     }
 
     return (
       <Dropdown
-        isOpen={this.state.dropdownOpen}
```

Comment:
What replaced these two props on the old Dropdown?  `isOpen` and `toggle`?  Can we remove `this.state.dropdownOpen` and `this.toggleDropdown` now?

Reply:
Removed!